src/Illuminate/Validation/Rules/OrRule.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Illuminate\Validation\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Str;
+
+class OrRule implements Rule
```

Comment:
The Rule contract is deprecated. Do you mind of implementing Illuminate\Contracts\Validation\ValidationRule instead? Sorry for not mentioning this at first.

Reply:
No problem